<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changepassemp</class>
 <widget class="QDialog" name="changepassemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_cp">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>150</y>
     <width>481</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_cpwd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_curpwdemp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newpwdemp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cfnewpwdemp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpwd_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setpassemp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
